src/types/types.ts

```diff
@@ -2591,6 +2591,93 @@ export interface DebugPrestate {
   storage: Record<string, string>;
 }
 
+/*
+ * SDK Representation of a user operation and bundler types
+ */
+export interface UserOperation {
```

Comment:
can we have UserOperation extend UserOperationRequest? Should be good to use inheritance here since they are all interfaces.